### PR TITLE
Send email to approver and creator

### PR DESCRIPTION
**Pull request by JasmeetKaur-Defra**

### CODE REVIEW

File: RPA.MIT.Notification.Function/Validation/ValidateMessage.cs [hunk 1]
```diff
@@ -21,7 +21,10 @@ public static class ValidateMessage
             ""Scheme"": {
                 ""type"": ""string""
             },
-            ""EmailRecipient"": {
+            ""ApproverEmail"": {
```
Comment (andygibbonsdefra) [inline RPA.MIT.Notification.Function/Validation/ValidateMessage.cs:24]: This should still be just EmailRecipient? (Just one generic email, not 2 specific Approver/Creator emails...

File: RPA.MIT.Notification.Function/Validation/ValidateMessage.cs [hunk 2]
```diff
@@ -30,7 +33,8 @@ public static class ValidateMessage
             ""Data"",
             ""Id"",
             ""Scheme"",
-            ""EmailRecipient""
+            ""ApproverEmail"",
```
Comment (andygibbonsdefra) [inline RPA.MIT.Notification.Function/Validation/ValidateMessage.cs:36]: Same thing here. Should just be 'EmailRecipient'

**Review by andygibbonsdefra — CHANGES_REQUESTED** [review]
Need to revert the CreatorEmail and ApproverEmail to just be EmailRecipient.

File: RPA.MIT.Notification.Function/Validation/ValidateMessage.cs (hunk 1 above)
Comment (JasmeetKaur-Defra) [reply RPA.MIT.Notification.Function/Validation/ValidateMessage.cs]: Reverted the changes

File: RPA.MIT.Notification.Function/Validation/ValidateMessage.cs (hunk 2 above)
Comment (JasmeetKaur-Defra) [reply RPA.MIT.Notification.Function/Validation/ValidateMessage.cs]: Reverted the changes

**Comment by andygibbonsdefra** [comment]
I think this is OK now, but will hold off approval until the Web code is sorted.

**Review by andygibbonsdefra — APPROVED** [review]

**Pull request merged**